funtofem/model/function.py

```diff
@@ -267,48 +295,33 @@ def test_struct(cls):
         return cls(name="struct-func", analysis_type="structural")
 
     @classmethod
-    def temperature(cls, start: int = 0, stop: int = -1, body: int = -1):
+    def temperature(cls):
         """
         Class constructor for the Temperature function
         """
-        return cls(
-            name="temperature",
-            analysis_type="structural",
-            start=start,
-            stop=stop,
-            body=body,
-        )
+        return cls(name="temperature", analysis_type="structural")
 
     @classmethod
-    def xcom(cls, start: int = 0, stop: int = -1, body: int = -1):
+    def xcom(cls):
```

Comment:
What about changing xcom, ycom, zcom to x_cg, ... or xcg ? When I see com, I think communicator. Just a thought. The name="xcom" that is returned would stay unchanged.

Reply:
I think I'm going to change it to `Function.center_of_gravity(direc="x")` instead of three class methods which are very similar